internal/host/validator.go

```diff
@@ -600,7 +600,7 @@ func (v *validator) belongsToMachineCidr(c *validationContext) (ValidationStatus
 	if c.inventory == nil || !network.IsMachineCidrAvailable(c.cluster) {
 		return ValidationPending, "Missing inventory or machine network CIDR"
 	}
-	if !network.IsHostInPrimaryMachineNetCidr(v.log, c.cluster, c.host) {
+	if !network.IsHostInMachineNetCidrs(v.log, c.cluster, c.host) {
 		return ValidationFailure, "Host does not belong to machine network CIDRs. Verify that the host belongs to every CIDR listed under machine networks"
```

Comment:
This error message is not true now.  Right ?

Reply:
Ah yep, last part is not.

Reply:
Wording is awkward here, because we still have to belong to both IPv4 and IPv6 CIDRs if both appear in the list. What do you think of "Verify that the host belongs to a CIDR for every IP version listed under machine networks"?

Reply:
That might be OK.  @avishayt what do you think?

Reply:
"Verify that the host belongs to a machine network CIDR for every used IP version"?

Reply:
Changed to "Verify that the host belongs to a machine network CIDR listed for each IP stack in use"